app/src/lib/menu-update.ts

```diff
@@ -181,13 +182,15 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     }
 
     menuStateBuilder.disable('view-repository-on-github')
+    menuStateBuilder.disable('create-pull-request')
 
     if (
       selectedState &&
       selectedState.type === SelectionType.MissingRepository
     ) {
       if (selectedState.repository.gitHubRepository) {
         menuStateBuilder.enable('view-repository-on-github')
+        menuStateBuilder.enable('create-pull-request')
```

Comment:
Does that mean `menuStateBuilder.enable('view-repository-on-github')` is wrong too?